{
  "id": "BIT-moodle-2022-40314",
  "details": "A remote code execution risk when restoring backup files originating from Moodle 1.9 was identified.",
  "aliases": [
    "CVE-2022-40314",
    "GHSA-2hmm-q272-xmhf"
  ],
  "modified": "2024-04-23T23:58:47.114887Z",
  "published": "2024-03-06T11:02:48.263Z",
  "database_specific": {
    "cpes": [
      "cpe:2.3:a:moodle:moodle:*:*:*:*:*:*:*:*"
    ],
    "severity": "Critical"
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2128147"
    },
    {
      "type": "WEB",
      "url": "https://moodle.org/mod/forum/discuss.php?d=438393"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "moodle",
        "ecosystem": "Bitnami",
        "purl": "pkg:bitnami/moodle"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "3.9.0"
            },
            {
              "fixed": "3.9.17"
            },
            {
              "introduced": "3.11.0"
            },
            {
              "fixed": "3.11.10"
            },
            {
              "introduced": "4.0.0"
            },
            {
              "fixed": "4.0.4"
            }
          ]
        }
      ],
      "database_specific": {
        "source": "https://github.com/bitnami/vulndb/tree/main/data/moodle/BIT-moodle-2022-40314.json"
      },
      "severity": [
        {
          "type": "CVSS_V3",
          "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
        }
      ]
    }
  ],
  "schema_version": "1.6.0"
}